common/services/prismic/link-resolver.ts

```diff
@@ -1,46 +1,47 @@
 import { isContentType } from './content-types';
 
 type Props = {
-  id: string;
+  uid?: string;
   type: string;
 };
 type DataProps = {
-  id: string;
+  uid?: string;
   type: string;
   data: {
     relatedDocument?: {
-      id: string;
+      uid: string;
       type: string;
     };
   };
 };
-function linkResolver(doc: Props | DataProps): string {
-  const { id, type } = doc;
 
-  if (type === 'webcomics') return `/articles/${id}`;
-  if (type === 'webcomic-series') return `/series/${id}`;
+function linkResolver(doc: Props | DataProps): string {
+  const { uid, type } = doc;
+  if (!uid) return '/';
+  if (type === 'webcomics') return `/articles/${uid}`;
+  if (type === 'webcomic-series') return `/series/${uid}`;
   if (
     type === 'exhibition-guides' ||
     type === 'exhibition-texts' ||
     type === 'exhibition-highlight-tours'
   )
-    return `/guides/exhibitions/${id}`;
+    return `/guides/exhibitions/${uid}`;
```

Comment:
This is kind of unrelated to this ticket, but am I correct in thinking this could potentially link to a 404 since it only uses one of the guides' id?

Reply:
Not sure I understand what you mean.

It uses the uid of whichever one of these has matched, so it shouldn't 404

Reply:
I'll be honest I can't fully remember 😅 
I think I meant that the condition might be true but return a 404 because if the `id/uid` that's passed in is an `exhibition-text`'s, but the guide uses the `exhibition-highlight-tours`'s `id`/`uid` in the URL. I guess, though, that's if it happens it's because _we've_ used the wrong one, so it's a non-issue.
Happy to resolve this.

Reply:
I don't think it'll 404 because [we try and get all the different types as we don't know which one it is](https://github.com/wellcomecollection/wellcomecollection.org/blob/9f0f78b8420dee43b5d2e7baa54b392298bfeb9b/content/webapp/pages/guides/exhibitions/%5Bid%5D/index.tsx#L95), but this has got me thinking what exactly we link to from /guides/exhibitions to an exhibition guide page - I think it'll work with whatever is used as a uid for either the exhibition-text or the exhibition-higlight-tour, but we would want them to be the same